modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc

```diff
@@ -331,20 +331,63 @@ As the Helm chart has currently not been published to a Helm repository, you nee
 minikube start --kubernetes-version=v1.28.1
 ----
 
-. Enable the minikube ingress plugin, which acts like a reverse proxy for your cluster:
+. Enable the `minikube ingress` plugin, which acts like a reverse proxy for your cluster:
 +
 [source,bash]
 ----
 minikube addons enable ingress
 ----
 
+. Linux only: enable the `ingress-dns` plugin and configure it:
++
+[source,bash]
+----
+minikube addons enable ingress-dns
+----
+
+. Configure the `in-cluster DNS server` to resolve local DNS names inside the cluster`:
++
+Note that this step is not optional but mandatory for an Infinite Scale installation. For details see https://minikube.sigs.k8s.io/docs/handbook/addons/ingress-dns/#installation[Step 4, (optional) Configure in-cluster DNS server to resolve local DNS names inside cluster].
+
+. macOS only: run `minikube tunnel` to be expose `80,443` ports:
++
+[source,bash]
+----
+minikube tunnel
+----
+
+. Configure hosts:
+.. On Linux you need to add additional configurations to use the ingress by adding the domain names to `/etc/hosts`. Those entries need to point to the Minikube interface IP which you can get by running `minikube ip`.
++
+[source,bash]
+----
+192.168.49.2 ocis.kube.owncloud.test
+----
+
+.. On macOS you need to add additional configurations to use the ingress by adding the domain names to `/etc/hosts`. Since you are using `minikube tunnel`, those entries need to point to `127.0.0.1` because it's listening on the localhost interface.
```

Comment:
```suggestion
.. On macOS you need to add additional configurations to use ingress by adding the domain names to `/etc/hosts`. Since you are using `minikube tunnel`, those entries need to point to `127.0.0.1` because it's listening on the localhost interface.
```